content/en/profiler/enabling/java.md

```diff
@@ -272,12 +272,10 @@ The allocation profiler engine does not depend on the `/proc/sys/kernel/perf_eve
 
 {{< /tabs >}}
 
-### Live-heap profiler engine (beta)
+### Live-heap profiler engine
```

Comment:
We were not really planning to make this mode 'GA' immediately at this moment.
Currently, it is available with a word of caution because we had several false starts in the past and we want to run it without total commitment for a longer time before we claim it GA with all the consequences in support etc.

Effectively, this would fall under 'Preview' but IIUC that would assume gated access which is not really true, because the way to enable the feature has been in public doc for a while.

And last, we might YOLO this and just do a silent GA, considering that we already have a non-trivial number of customers enabling it in the 'beta' mode (cca. last 2-3 months) and we haven't been receiving an increased number of escalations related to this feature. We would be just making the de-facto state to be de-jure.

Ok, after a lot of words - my point is, we could go with this particular change but for the future we need to figure out how to deal with the library lifecycle which is not completely compatible with the backend lifecycle.